integration-tests/actions/vrf/vrfv2plus/setup_steps.go

```diff
@@ -354,20 +331,28 @@ func SetupVRFV2PlusWrapperEnvironment(
 	if err != nil {
 		return nil, nil, err
 	}
-	err = evmClient.WaitForEvents()
-	if err != nil {
-		return nil, nil, fmt.Errorf(vrfcommon.ErrGenericFormat, vrfcommon.ErrWaitTXsComplete, err)
-	}
 
 	//fund consumer with Eth
-	err = wrapperContracts.LoadTestConsumers[0].Fund(big.NewFloat(*vrfv2PlusConfig.WrapperConsumerFundingAmountNativeToken))
+	gasLimit := int64(2 * 21000)
```

Comment:
I really dont like this approach, it should be something more flexible

Reply:
@iljapavlovs fixed in https://github.com/smartcontractkit/chainlink/pull/12964/commits/dbe63f86f64215dc90dc9fe01cacedcd2823f7c9